src/lib/seam/connect/routes/workspaces.ts

```diff
@@ -0,0 +1,36 @@
+// TODO: This file is generated from route spec
+import type { Routes } from '@seamapi/types/connect'
+import type { Axios } from 'axios'
+import type { SetNonNullable } from 'type-fest'
+
+export class Workspaces {
```

Comment:
is the reasoning for making each namespace a separate class to more easily tree shake?

Reply:
Not at first, but I realized after I did this that it will enable opt-in tree shaking via individual imports and a slight change to what I have here.

The main reason is to isolate the generated code to a single folder / individual files and simplify code gen.

Reply:
Made some small adjustments that in theory would allow one to use each of the resource classes individually.